framework/web/Request.php

```diff
@@ -1008,9 +1007,17 @@ private function utf8Encode($s)
         $len = \strlen($s);
         for ($i = $len >> 1, $j = 0; $i < $len; ++$i, ++$j) {
             switch (true) {
-                case $s[$i] < "\x80": $s[$j] = $s[$i]; break;
-                case $s[$i] < "\xC0": $s[$j] = "\xC2"; $s[++$j] = $s[$i]; break;
-                default: $s[$j] = "\xC3"; $s[++$j] = \chr(\ord($s[$i]) - 64); break;
+                case $s[$i] < "\x80": $s[$j] = $s[$i];
+
+break;
+                case $s[$i] < "\xC0": $s[$j] = "\xC2";
+                    $s[++$j] = $s[$i];
+
+break;
+                default: $s[$j] = "\xC3";
```

Comment:
Shouldn't it be on the next line?

Reply:
I did not do any manual changes, only the automatic ones.

Reply:
we can prevent this for this function if we add those to the docblock:
```
     * @phpcs:disable Generic.Formatting.DisallowMultipleStatements.SameLine
     * @phpcs:disable Squiz.WhiteSpace.ScopeClosingBrace.ContentBefore
```
I guess this is ok, right? it looks ugly if we try to refactor it.

Reply:
How about excluding these rules?

Reply:
Did you tried to fix it manually? Maybe fixer is not able to fix it correctly, but it won't mess with code style if there is nothing to fix.

Reply:
Ignored this function in the docblock.